### PR TITLE
Fix protobuf signature signing

### PR DESCRIPTION
**Pull request by bb111189**
## Description

**LookupSetDSBlockFromSeed**
- sig should be sig(lowBlockNum, highBlockNum, dsBlocks) instead of sig (dsBlocks)

**LookupSetTxBlockFromSeed**
- sig should be sig(lowBlockNum, highBlockNum, txblocks) instead of sig(txblocks)

**LookupSetStateDeltaFromSeed**
- sig should be sig(blockNum, stateDelta) instead of sig(stateDelta)

**Refactoring of VCNodeSetDSTxBlockFromSeed**

### Implementation

- [x] **ready for review**

### Integration Test (Core Team)

- [ ] ~local machine test~

- [x] medium-scale cloud test (commit: 0b467b79b60dfa7d6d028c017e9ff59445148ee9)

### CODE REVIEW

**Comment by ShengguangXiao** [comment]
@bb111189 I make a change in this PR, directly return false if protobuf data is not initialized, please sync your code and do a double check

File: src/libMessage/Messenger.cpp [hunk 1]
```diff
@@ -3326,22 +3325,22 @@ bool Messenger::SetDSPoWSubmission(
...
-      return false;
-    }
+  bytes tmp(result.data().ByteSize());
+  result.data().SerializeToArray(tmp.data(), tmp.size());
 
-    SerializableToProtobufByteArray(signature, *result.mutable_signature());
-  } else {
-    LOG_GENERAL(WARNING, "DSPoWSubmission.Data initialization failed.");
+  Signature signature;
+  if (!MultiSig::GetInstance().SignKey(tmp, submitterKey, signature)) {
```
Comment (bb111189) [inline src/libMessage/Messenger.cpp]: To look into this

Comment (ShengguangXiao) [reply src/libMessage/Messenger.cpp]: @sandipbhoir Why it use MultiSig here? It is not using aggregated pubkey.

Comment (sandipbhoir) [reply src/libMessage/Messenger.cpp]: I didn’t use multisig. As with my commit-   af700ad , Schnorr was used. It must have modified in following pr’s for some reason.

Comment (ansnunez) [reply src/libMessage/Messenger.cpp]: This one is okay, please refer to #1097 description for the explanation why we use Multisig

**Pull request merged**